_pages/getting-started/equipment.md

```diff
@@ -27,6 +27,16 @@ Your laptop will come preloaded with the latest, GSA-approved/supported Mac oper
 
 **Encrypt all portable media that will connect to any TTS system.** You can apply FileVault encryption to removable USB drives and external hard drives by right-clicking the drive on your desktop and clicking **Encrypt**. You must choose a strong password you can remember, as your drive will require the password to mount and retrieve data once encrypted.
 
+### Bring Your Own Device (BYOD)
+
+GSA has implemented a BYOD policy that allows users to connect their non-GSA procured devices, which have been previously approved by IT security, to GSA resources in a native fashion, utilizing the GSA network. Please note, only approved smartphones and tablets can be connected to the GSA network. Personal laptops cannot be connected directly to the GSA network at any GSA facility. For more information, please reference Section 4.2 of the [IT Security Procedural Guide](https://www.gsa.gov/cdnstatic/Securing_Mobile_Devices_and_Applications_%5BCIO_IT_Security_12-67_Rev_4%5D_01-26-2018.pdf#page=11).
+
+For users who work on a FISMA system that wish to use their personal laptop/computer (at a non-GSA facility), the assigned ISSO/ISSM for the application must accept the associated risk(s), obtain approval from the Authorizing Official, and document the use of the personally owned computer within the System Security Plan (SSP).
+
+For users who do not work directly on a FISMA system who wish to use their personal machine (at a non-GSA facility), approval will be granted on a case-by-case basis by GSA IT-Security. Post your questions in the [AskIT Chatter Page](https://gsa.my.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000001I35) for guidance. **Controlled Unclassified Information (CUI) can never be stored on a personal machine and the best approach is to utilize the Government Furnised Equipment (GFE) to the maximum extent possible.**
+
+For additional sofware needs, please visit [this page](https://handbook.tts.gsa.gov/software/).
```

Comment:
I don't think this part is relevant here.

```suggestion
```

Reply:
I thought it to be relevant since I'm assuming folks want to use their personal machines to use software/tools not available of their GFE. If thats the case, they should work towards getting the tools they need on their GFE vs using their own device.

Reply:
Hmm. I don't think folks should be installing software GSA/TTS pays for on their personal machines. I worry this could imply that we would.

Reply:
Ah, I see what you're saying now. Good catch.